main/tasks/asic_task.c

```diff
@@ -28,6 +28,8 @@ void ASIC_task(void *pvParameters)
         GLOBAL_STATE->valid_jobs[i] = 0;
     }
 
+    GLOBAL_STATE->job_queue_initalized = true;
+
     ESP_LOGI(TAG, "ASIC Job Interval: %.2f ms", GLOBAL_STATE->asic_job_frequency_ms);
     SYSTEM_notify_mining_started(GLOBAL_STATE);
```

Comment:
No need for a new boolean, you can leverage `duration_start` which is set in `SYSTEM_notify_mining_started`.